nflverse/nflfastR
# add gsis_id patch

Test instructions:

```r
usethis::pr_fetch(379) # check out this PR
devtools::load_all() # load devel version
nflfastR::build_nflfastR_pbp("2022_02_CHI_GB")
# should see a new "patching gsis_ids" thing. 
```
